core/modules/modPrune.class.php

```diff
@@ -223,10 +223,9 @@ public function init($options = '')
 	 * @param   string  $options    Options when enabling module ('', 'noboxes')
 	 * @return  int                 1 if OK, 0 if KO
 	 */
-	public function remove($options = '')
-	{
+	public function remove($options= '') {
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: Squiz.Functions.FunctionDeclarationArgumentSpacing.SpaceBeforeEquals
**Severity**: ERROR
**File**: core/modules/modPrune.class.php L226

Incorrect spacing between argument &quot;$options&quot; and equals sign; expected 1 but found 0

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1693651003>*